cmd/root.go

```diff
@@ -124,3 +124,19 @@ func extractBucketAndPrefix(input string) (string, string, error) {
 func SetVersionInfo(version, date string) {
 	rootCmd.Version = fmt.Sprintf("%s (Built on %s)", version, date)
 }
+
+func defaultProfile() string {
+	if p, ok := os.LookupEnv("AWS_PROFILE"); ok {
+		return p
+	}
+	return "local"
```

Comment:
`local` is the current default value, but the default value of `AWS_PROFILE` is `default` in AWS CLI.

Reply:
@orangekame3 fixed! https://github.com/orangekame3/stree/pull/19/commits/1be1b8c8f9e31c92bc20f0b1770176c4f3fd866f

Reply:
@fujiwara 
Thank you! :100: